.github/workflows/notify-docs-triager.yml

```diff
@@ -0,0 +1,29 @@
+name: Notify Docs-triagers
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+
+jobs:
+  Notify-triagers:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v4.1.1
+
+      - name: Check PR Changes
+        id: pr-changes
+        uses: tj-actions/changed-files@v42
+        with:
+          files: |
+            **.md
+
+      - name: Create Comment
+        if: steps.pr-changes.outputs.any_changed == 'true'
+        uses: mshick/add-pr-comment@v2.8.2
+        with:
+          message: "The PR contains modifications to `.md` files, hence tagging Docs triagers for review and approval.
+                    cc: @alequetzalli @TRohit20 @octonawish-akcodes @BhaswatiRoy @VaishnaviNandakumar @Arya-Gupta @J0SAL"
```

Comment:
Hey @TRohit20, we are not creating workflow only to trigger the Doc triagers. Instead it should work for all the triager maintainers according to the changes made inside the PR.